CHANGELOG.md

```diff
@@ -3,6 +3,116 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.0
+
+_Sep 16, 2021_
+
+🎉 This is the first release of the data grid component with full support of the new MUI 5.0 release 🎉!
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡ Migrate to the new `@mui/material` and `@mui/styles` packages and drop support for `@material-ui/core` (#2515, #2571) @m4theushw
+- 📚 Migrate to the new documentation infrastructure and design (#2441) (@DanailH, @m4theushw)
+- 🚀 Introduce the row menu feature  (#2385) @m4theushw
+- 🚀 Introduce the `disableVirtualization` prop (#2326) @m4theushw
+- 🚀 Introduce the new `isRowSelected` api method (#2523) @flaviendelangle
+- 🌎 Add Simplified Chinese (zhCN) localization (#2431) @wlf100220
+- 🌎 Add Korean (koKR) localization (#2446) @zzossig
+
+### `@mui/x-data-grid@v5.0.0-beta.0` / `@mui/x-data-grid-pro@v5.0.0-beta.0`
+
+#### Breaking changes
+
+- [DataGridPro] Remove `apiRef.current.getState` method
+
+  ```diff
+  -const state = apiRef.current.getState();
+  +const state = apiRef.current.state
+  ```
+
+- [DataGridPro] Unification of the selection API parameters between `selectRow` and `selectRows`
+
+
+  ```diff
+  -selectRow: (id: GridRowId, isSelected?: boolean, allowMultiple?: boolean = false) => void;
+  +selectRow: (id: GridRowId, isSelected?: boolean, resetSelection?: boolean = false) => void;
+  ```
+
+- [DataGrid] Replace the `options` with direct prop reads
+
+  ```diff
+  -const { options } = useGridSlotComponentProps();
+  +const rootProps = useGridRootProps();
+  ``` 
+
+#### Changes
+
+- [DataGridPro] Only apply checkboxSelectionVisibleOnly when pagination is enabled (#2443) @flaviendelangle
```

Comment:
Can you check the other places where we have code like text and wrap it in `` ;)

```suggestion
- [DataGridPro] Only apply `checkboxSelectionVisibleOnly` when pagination is enabled (#2443) @flaviendelangle
```

Reply:
Yes of course

Reply:
Done